phy/cluster/manual/_history.py

```diff
@@ -125,18 +131,18 @@ def __init__(self, process_ups=None):
         self.process_ups = process_ups
 
     def action(self, *controllers):
-        """Register one or several controllers for this action.
-
-        The order matters: the first controller does the first action, etc.
```

Comment:
indeed. I thought I would be using this feature but it looks I won't... hopefully this bit can be simplified at some point..